src/coreclr/src/System.Private.CoreLib/src/System/Math.CoreCLR.cs

```diff
@@ -11,117 +11,149 @@
 ===========================================================*/
 
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 
 namespace System
 {
     public static partial class Math
     {
+#pragma warning disable CA1401 // P/Invokes should not be visible
         [Intrinsic]
```

Comment:
I am not sure whether these methods are going be recognize as `CORINFO_INTRINSIC_...` intrinsics once the methods are switched to DllImport. This may need updating these intrinsics to the new plan where the JIT recognizes them by name.